sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/query.py

```diff
@@ -57,37 +59,24 @@ def cinder_addon_decisions_download(date, bearer_token):
     return response
 
 
-def check_json(cinder_addon_decisions_response_text):
-    """Script will return an empty dictionary for apps on days when there is no data. Check for that here."""
-    with tempfile.NamedTemporaryFile() as tmp_json:
-        with open(tmp_json.name, "w") as f_json:
-            f_json.write(cinder_addon_decisions_response_text)
-            try:
-                query_export = read_json(f_json.name)
-            except (
-                ValueError
-            ):  # ex. json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
-                return None
-    return query_export
-
-
 def add_date_to_json(query_export_contents, date):
     """Add a date to the entries so we can partition table by this date."""
     fields_list = []
     for item in query_export_contents:
         field_dict = {
             "date": date,
-            "decision_type": item["decision_type"],
-            "user": item["user"],
-            "created_at": item["created_at"],
-            "entity_id": item["entity_id"],
-            "entity": item["entity"],
-            "uuid": item["uuid"],
-            "entity_slug": item["entity_slug"],
-            "job_id": item["job_id"],
-            "job_assigned_at": item["job_assigned_at"],
-            "typed_metadata": item["typed_metadata"],
-            "applied_policies": item["applied_policies"],
+            "decision_type": str(item["decision_type"]),
+            "user": str(item["user"]),
+            "created_at": (item["created_at"]),
+            "entity_id": str(item["entity_id"]),
+            "entity": (item["entity"]),
+            "uuid": str(item["uuid"]),
+            "entity_slug": str(item["entity_slug"]),
+            "job_id": str(item["job_id"]),
+            "job_assigned_at": (item["job_assigned_at"]),
+            "queue_slug": str(item["queue_slug"]),
+            "typed_metadata": (item["typed_metadata"]),
+            "applied_policies": (item["applied_policies"]),
```

Comment:
I'd suggest removing `str()` for all these, plus the extra leftover parentheses.
```suggestion
            "decision_type": item["decision_type"],
            "user": item["user"],
            "created_at": item["created_at"],
            "entity_id": item["entity_id"],
            "entity": item["entity"],
            "uuid": item["uuid"],
            "entity_slug": item["entity_slug"],
            "job_id": item["job_id"],
            "job_assigned_at": item["job_assigned_at"],
            "queue_slug": item["queue_slug"],
            "typed_metadata": item["typed_metadata"],
            "applied_policies": item["applied_policies"],
```

Otherwise you run the risk of `str()` converting nulls into the string "None".